app-config.yaml

```diff
@@ -298,6 +298,14 @@ scaffolder:
   #   email: scaffolder@backstage.io
   # Use to customize the default commit message when new components are created
   # defaultCommitMessage: 'Initial commit'
+  #
+  # Use to run check on stale tasks every minute. By default, it is '*/5 * * * *', what means to run every 5 minutes.
+  # staleTasks:
+  #   cron: * * * * *
+  #
+  # Use to consider the task stale when no task heartbeat during 2 minutes. By default, it is 24 hours.
+  # staleTasks:
+  #   timeout: 120 # seconds
```

Comment:
You can remove this comment I think. Duration specs are well understood already, and this comment won't be visible to any adopters anyway (this is the in-repo config only, so it doesn't help those who scaffold their own repo).